curl.yaml

```diff
@@ -30,26 +28,17 @@ environment:
       - openssl-dev
       - zlib-dev
       - brotli-dev
-
 pipeline:
   - uses: fetch
     with:
       uri: https://curl.se/download/curl-${{package.version}}.tar.xz
       expected-sha256: ee5f1a1955b0ed413435ef79db28b834ea5f0fb7c8cfb1ce47175cc3bee08fff
   - uses: autoconf/configure
     with:
-      opts: |
-        --enable-ipv6 \
-        --enable-unix-sockets \
-        --with-openssl \
-        --with-nghttp2 \
-        --with-pic \
-        --disable-ldap \
-        --without-libssh2    
+      opts: "--enable-ipv6 \\\n--enable-unix-sockets \\\n--with-openssl \\\n--with-nghttp2 \\\n--with-pic \\\n--disable-ldap \\\n--without-libssh2    \n"
```

Comment:
this is suffering from the same issues some `melange bump` calls make.  I think @kaniini suggested we could try and set the format of more nodes to https://github.com/chainguard-dev/melange/blob/bd4e1298b6f461ffcd203b2b4e0bb29107ecc6e6/pkg/renovate/bump/bump.go#L76

I did have a quick try of that but it didn't seem to work.  Maybe I was doing it wrong though.